examples/angular/src/pages/ui/components/authenticator/custom-slots/custom-slots.component.ts

```diff
@@ -9,7 +9,7 @@ import awsExports from '@environments/auth-with-federated/src/aws-exports';
   templateUrl: 'custom-slots.component.html',
 })
 export class CustomSlotsComponent {
-  constructor() {
+  constructor(private authenticator: AuthenticatorService) {
```

Comment:
Want this back to public?

Reply:
I'll approve and you can make that change

Reply:
Oh right, good catch!

Reply:
```suggestion
  constructor(public authenticator: AuthenticatorService) {
```